.github/workflows/vhiveease_golang.yml

```diff
@@ -0,0 +1,42 @@
+name: Build and deploy vhive golang image
```

Comment:
Need a more general name. Golang is not the only language. also the name of the workflow should be shorter. 
Also, rename the file to image_build.yml